css/Components/Root.scss

```diff
@@ -0,0 +1,5 @@
+@import "../Theme/Color";
+
+body {
+  --bg-color: $Color-deep-cove;
```

Comment:
We could probably rename this variable into `Color-bg-color`; Or maybe we can move all of this into `Layout` and use `Layout-bg-color`;